crates/anvil/src/eth/api.rs

```diff
@@ -1067,6 +1070,44 @@ impl EthApi {
         self.add_pending_transaction(pending_transaction, requires, provides)
     }
 
+    /// Sends signed transaction, returning its receipt.
+    ///
+    /// Handler for ETH RPC call: `eth_sendRawTransactionSync`
+    pub async fn send_raw_transaction_sync(&self, tx: Bytes) -> Result<ReceiptResponse> {
+        node_info!("eth_sendRawTransactionSync");
+
+        let this = self.clone();
+        let hash = this.send_raw_transaction(tx).await?;
+
+        let mut stream = this.new_block_notifications();
```

Comment:
can we move this before the send_raw call, so that we can prevent race conditions here